R/pkg/R/sparkR.R

```diff
@@ -365,6 +365,10 @@ sparkR.session <- function(
     }
     overrideEnvs(sparkConfigMap, paramMap)
   }
+  if (nzchar(master)) {
+    assign("spark.master", master, envir = sparkConfigMap)
```

Comment:
It is - we are turning the Scala Builder syntax of multiple function calls into one in R; we just pick the order that most prominently featured which is 
```
SparkSession.builder().master("yarn").config("spark.master", "local[3]")
```